ipython/generateResonanceStructures.ipynb

```diff
@@ -0,0 +1,236 @@
+    "                print '\\n'\n",
+    "    elif dictionary:\n",
+    "        speciesDict = chemkin.loadSpeciesDictionary(dictionary)\n",
+    "        i=0\n",
+    "        for spc in speciesDict.iteritems():\n",
+    "            print \"\\n\\n**********************\\n\"\n",
+    "            print \"species {1} has {0} resonance structures:\".format(len(spc[1].molecule),spc[0])\n",
+    "            for struc in spc[1].molecule:\n",
+    "                display(struc)\n",
+    "                print struc, '\\n', struc.toAdjacencyList()\n",
+    "                if showOctet: octetDeviation(struc)\n",
+    "                if not struc.reactive:\n",
+    "                    print \"reactive = {0}\".format(struc.reactive)\n",
+    "                for atom in struc.vertices:\n",
+    "                    if atom.radicalElectrons:\n",
+    "                        print \"atom.sortingLabel: {0}\".format(atom.sortingLabel)\n",
+    "                print '\\n'\n",
+    "            i+=1\n",
+    "        print i\n",
+    "            "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Input either SMILES, AdjList, a list of SMILES, or an RMG species dictionary, and run the appropriate resonate function call:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "original molecule: [CH2]C=O\n"
+     ]
+    },
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAEMAAAAZCAIAAADlvQS1AAAABmJLR0QA/wD/AP+gvaeTAAADqUlE\nQVRYhd2YT0jbUBzHX2VuZKWSmtB2soN/wGKFzlpQpChIFaSReRp4EQZjiJdimZBDT4KTHAseXA5F\nxLIi9SBFT7WgYkVklYk6dVgQUbSS2uJoamv17ZCZxvpnmU3d2OfU3/e993u/b/L6S1oZhBD8FxT8\nxb2dTmckEpEq219zkkgkNjc3dTrd0NBQOp2WICN8dFiW7e/vx3H84OBga2vLYrFotdrp6ekc0z62\nE6/XW1ZWVlNTQ5Lk2dkZJ/p8Pp1O19LSsrGx8eDMj+dkZWWlqanJYDDMzc3t7u52dHRUVFRMTk5y\no6lUyuFw4DhutVpjsdgD8j+GE4ZhrFarSqVyOBzpdJrX/X6/Xq9vbm5eXV29f6YY8uvkt1f64uJi\ndHRUrVZ3dXUdHx9zYjAYbGxsrK2tnZ+fF79XHp3wp399ff3+mScnJyRJqtVqiqKSySQner3e0tLS\n9vb2cDgsZru8ONne3iYIorKycmpqSvyqm32MZVmXyyXymF1zUl1dPTY2xodut1ur1d61cmJior6+\nHkEQHMctFksgEOD0nZ2drKv7R/h8vqqqqtv72OfP8NUr+OwZVKng+/cwEpHACU3TCoWCpun9/f1o\nNOrxeLq7u7mh8/NzkefhLvhvF0mSGfXTJ4ii0O2GsRj89g22tUGDAV718Qc6YVkWRVGapoXi5eVl\nLtXfhGEYu93+K0gkoFIJnc7McDwONRooqOEhbyvBYDAWi3V2dgpFmUwmwRuHAAzDBgYGfgVfvoBo\nFLx5kxl+/hy8fg18Pl54krW+p6ent7eX+5xKpUpKSm7uEYlE5HJ5UVGRpJXfC8MAuRwoFNfEFy/A\n9+98lH1PBgcHv15BURQnLiwsNDQ0mEwmm80GAMAwLB6Pn56e5rd6ITgO4nHw48c18fAQ4DgfZTtR\nKpUvryguLubE8vLy2dnZQCCwt7e3trZmNBpRFB0fHxcuhHn9nWM0AhQFHk9GYVng9QKzmReyT9et\n8GessLCwoKAAQRCKovr6+mQyGUEQCIL4/f6ZmZnh4WFJyxeAIODjR/DhA5DLQVsbODwENhtQqcDb\nt5k5wnZxf+9aXl62WCx86PF46urquOcJQRCLi4vS9q5bcLmgXg+fPoU4Dt+9gwwjHBT7jA+HwyaT\n6ejoKA8FSoMoJ8lksrW1NRgM5ruaXBDlZGRkRKPRmM1ms9m8tLTE66FQKBQK5a22P0MGc+g5GIYB\nACT8VyEXRPWuu7Db7VLVkTs53ZN/ip+ygYFFORXIAQAAAABJRU5ErkJggg==\n",
```

Comment:
I'm clearing the output cells.
I don't mind not including this notebook (tell me what you think, I'm OK with both ways). I think we previously discussed adding a resonance generation feature to the website, let's discuss it and decide whether or not to ha a filtration switch (or filter by default). It could be an excellent replacement for this notebook.